.github/workflows/api_refs.yaml

```
@@ -0,0 +1,54 @@
+name: 'Build API Refs'
+
+on:
+    workflow_dispatch:
+        inputs:
+            version:
+                description: 'Released version'
+                required: true
+                type: string
```

Not used to build the references, only to create the branch.

- The script could evolve to receive version as an argument instead of alway building for the latest version (Is there a need for building old references?).
- Or just rename the inputs being renamed to avoid ambiguity (my favorite).

```suggestion
            branch_suffix:
                description: "Used in branch name to open PR for the latest version's references"
                required: true
                type: string
```

---

Wrong, it's used to find the target branch